docu/Concepts/Snippets/1-Blockchain/Connector/protobuf.md

```diff
@@ -0,0 +1,12 @@
+Transaktionen werden als verschachtelte Objekte beschrieben
+Die Transaktion wandert als Objekt durch die verschiedenen Teilsysteme.
+Sie wird als serialisiertes Objekt gespeichert und übers Netz übertragen.
+Kurz gesagt heißt die Verwendung von Protobuf:
+Objektbasierte Programmierung auf Plattformübergreifende Art. 
```

Comment:
`
Protocol buffers are Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data – think XML, but smaller, faster, and simpler. You define how you want your data to be structured once, then you can use special generated source code to easily write and read your structured data to and from a variety of data streams and using a variety of languages.
`
Source: https://developers.google.com/protocol-buffers/

It is not a programming language, but a data transport format optimized for minimal transport size.

Reply:
Ja und ich bekomme den Datensatz in vielen Programmiersprachen direkt als Objekt geliefert mit dem bequem arbeiten kann.